CppExport/src/visitors/source_visitors/DeclarationVisitorSource.cpp

```diff
@@ -217,20 +184,25 @@ SourceFragment* DeclarationVisitorSource::visit(Method* method)
 SourceFragment* DeclarationVisitorSource::visit(VariableDeclaration* vd)
 {
 	auto fragment = new CompositeFragment(vd);
-	*fragment << printAnnotationsAndModifiers(vd);
-	*fragment << expression(vd->typeExpression()) << " " << vd->nameNode();
 	if (vd->initialValue())
+	{
+		*fragment << expression(vd->typeExpression()) << " ";
```

Comment:
Why do we only print this if the variable has an initial value? What it if doesn't?